test-lib.sh

```diff
@@ -34,12 +34,17 @@ function ct_cleanup() {
 
     : "Stopping and removing container $container..."
     docker stop "$container"
-    exit_status=$(docker inspect -f '{{.State.ExitCode}}' "$container")
-    if [ "$exit_status" != "$EXPECTED_EXIT_CODE" ]; then
-      : "Dumping logs for $container"
-      docker logs "$container"
+
+    # Container doesn't removed by `docker stop` and exists
```

Comment:
```suggestion
    # Container has not been removed by `docker stop` and still exists
```
It is just a nitpick, but I had to reread the comment to understand it. This way it seems clearer to me.